docs/developers/index.md

```diff
@@ -1,21 +1,42 @@
+(contributing-index)=
 # Contributing
 
-Here you can find resources about the contributing workflow for both code to napari core and documentation of the project, as well as information for onboarding of new core developers.
+We welcome your contributions! Here you will find resources to help you contribute
+to napari.
 
-You can see the general direction for napari development and possible work plans in our current [Roadmap](../roadmaps/index.md).
+napari has a [Code of Conduct](napari-coc) that should be honored by everyone who participates in the `napari` community.
 
-## For contributors
+- [](contact): how to contact us.
+- [](roadmaps): convey the general direction for napari development and possible
+  work plans.
+- **Contributor guides**: resources on for contributors:
+  - [Contributing guide](napari-contributing)
+    - [Development installation](dev-installation)
+    - [Testing](napari-testing)
+    - [Translations](translations)
+    - [Performance](performance)
+      - [Profiling](profiling)
+      - [Benchmarks](napari-benchmarks)
+    - [Contributing documentation](docs_contributing_guide)
+  - [](architecture-index)
 
-- [Contributing guide](./contributing)
-- [Contributing documentation](./documentation/index)
-- [Translations](./translations)
-- [Testing](./testing)
-- [Profiling](./profiling)
-- [Benchmarks](./benchmarks)
+- **Core developer guides**: resources for core developers, including information for
+  onboarding new core developers:
+  - [Core Developer guide](./core_dev_guide)
+  - [Maintenance](./maintenance)
```

Comment:
```suggestion
  - [Maintenance](coredev/maintenance)
```
Or, better yet add a reference in maintenance.md?

Reply:
There's already a reference!